pcsx2/GS.cpp

```diff
@@ -436,5 +423,5 @@ void gsResetFrameSkip()
 void SaveStateBase::gsFreeze()
 {
 	FreezeMem(PS2MEM_GS, 0x2000);
-	Freeze(gsRegionMode);
+	Freeze(gsVideoMode);
```

Comment:
I didn't catch it the first time, but I think this breaks savestate compatibility.